src/index.js

```diff
@@ -1,39 +1,63 @@
 import './style.css';
+// import { setNewScore, getScores } from './fectch.js';
 import Scores from './scores.js';
+// import getGameId from './getGameId.js';
```

Comment:
- [ ] Same here, kindly do not leave unused code as a comment in your codebase. This will make your code much noisier and not professional, It is always a good practice to keep your code clean from such comments 🙃 😉